shared/set_configs/EMN.json

```diff
@@ -14,11 +15,87 @@
     ]
   },
   "SET_CORRECTIONS": [{
-    "match": {
-      "multiverseid": 414497
+      "match": {
+        "imageName": "brisela, voice of nightmares1"
+      },
+      "replace": {
+        "number": "15b",
+        "names": [
+          "Bruna, the Fading Light",
+          "Brisela, Voice of Nightmares"
```

Comment:
Please look at the changes that I made to the documentation.

Meld cards are both one card and two cards. For physical card purposes, color identity, numbering, and some other characteristics, they are two cards. This PR makes meld cards work as two separate cards.

Putting all three names on the melded card would be taking meld cards as one single card. This is doable but would require many more changes. "number" would need to be an array, color identity would need to be distinguished based on which half was being considered, and I suspect there are others.

Having one attribute of the card take melded cards as single entities while every other attribute takes them as separate entities is inconsistent and confusing. I will add a note to the changelog indicating that the behavior has changed.